package/yast2-services-manager.spec

```diff
@@ -24,16 +24,16 @@
 ######################################################################
 
 Name:           yast2-services-manager
-Version:        4.1.1
+Version:        4.1.2
 Release:        0
 BuildArch:      noarch
 
 BuildRoot:      %{_tmppath}/%{name}-build
 Source0:        %{name}-%{version}.tar.bz2
 
 Requires:       ruby
-# Yast2::SystemService#build
-Requires:       yast2 >= 4.0.82
+# Yast2::SystemService#found? as an array
+Requires:       yast2 >= 4.0.83
```

Comment:
It should depend on yast2 4.1.2, shouldn't it?

Reply:
Oh, yes, I forgot (it is 4.1.1). I will adapt it.

Reply:
Well, the comment is wrong. But the 4.0.83 looks right (the change was introduced in that version). The socket improvements should not affect this module.